uwblueprint/feeding-canadian-kids
# Create getASPNearLocation query

## Notion Ticket Link
https://www.notion.so/uwblueprintexecs/Dev-e3112e78136f49b4b042e9a0d9df9723?p=404d7d68a38940f4957272ecb7523075&pm=s


## Implementation Description
* Returns all users with the role of ASP within a given max distance of the requesting user
* Results are sorted in increasing order of distance, with an option to pass in offset and limit
* Along with each user, returns distance away from the requesting user (in KM)



## Steps To Test
1. Backend tests are currently not possible due to lack of support from mongomock, query should be tested in GraphQL playground
2. To verify that distance is accurate (to a certain degree), you can use this website: https://www.mapdevelopers.com/distance_from_to.php


## What Should Reviewers Focus On?
*


## Checklist
